android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -73,7 +73,18 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public void onDestroy() {
+        // Here we are trying to handle the following corner case: an application using the SDK
+        // is using this Activity for displaying meetings, but there is another "main" Activity
+        // with other content. If this Activity is "swiped out" from the recent list we will get
+        // Activity#onDestroy() called without warning. At this point we can try to leave the
+        // current meeting, but when our view is detached from React the JS <-> Native bridge won't
+        // be operational so the external API won't be able to notify the native side that the
+        // conference terminated. Thus, try our best to clean up.
         leave();
+        if (AudioModeModule.useConnectionService()) {
+            ConnectionService.abortConnections();
```

Comment:
Same story about the scope, doesn't care although it's my fault too, because I made the ConnectionService scopeless.